src/config/navigation.tsx

```diff
@@ -57,16 +76,16 @@ export const appScreensNavOptions = {
     headerShown: false
   },
   [Routes.Home]: {
-    headerTitle: 'Home'
+    headerShown: false
   }
 };
 
 export const statusBarStyles = {
   // TODO: Change these styles to customize the status bar
-  [Routes.Login]: statusBarConfig.whiteStatusBar,
-  [Routes.SignUp]: statusBarConfig.blueStatusBar,
-  [Routes.Tab1]: statusBarConfig.blueStatusBar,
-  [Routes.Tab2]: statusBarConfig.blueStatusBar,
-  [Routes.Home]: statusBarConfig.blueStatusBar,
+  [Routes.Login]: statusBarConfig.greenStatusBar,
```

Comment:
is not it better to be the default? greenStatusBar

Reply:
replace with default